internal/rootcoord/import_manager.go

```diff
@@ -155,7 +157,7 @@ func (m *importManager) sendOutTasks(ctx context.Context) error {
 		m.busyNodes[resp.GetDatanodeId()] = true
 
 		// erase this task from head of pending list if the callImportService succeed
-		m.pendingTasks = m.pendingTasks[1:]
```

Comment:
NIT: will have a lot of copy if `pendingTasks` is large. Maybe move last element to first and make length - 1 will do the trick.